src/Storage/custom/hybrid-2019-03-01/Update-AzStorageAccount.ps1

```diff
@@ -0,0 +1,373 @@
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Support.AccessTier]
+        # Required for storage accounts where kind = BlobStorage. The access tier used for billing.
+        ${AccessTier},
+    
+        [Parameter(ParameterSetName='UpdateExpanded', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateExpandedStorageEncryption', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateExpandedKeyVaultEncryption', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpanded', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedStorageEncryption', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedKeyVaultEncryption', HelpMessage='A boolean indicating whether or not the service encrypts the data as it is stored.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Body')]
+        [System.Management.Automation.SwitchParameter]
+        # A boolean indicating whether or not the service encrypts the data as it is stored.
+        ${BlobEnabled},
+    
+        [Parameter(ParameterSetName='UpdateExpanded', Mandatory, HelpMessage='Gets or sets the custom domain name assigned to the storage account. Name is the CNAME source.')]
+        [Parameter(ParameterSetName='UpdateExpandedStorageEncryption', Mandatory, HelpMessage='Gets or sets the custom domain name assigned to the storage account. Name is the CNAME source.')]
+        [Parameter(ParameterSetName='UpdateExpandedKeyVaultEncryption', Mandatory, HelpMessage='Gets or sets the custom domain name assigned to the storage account. Name is the CNAME source.')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpanded', Mandatory, HelpMessage='Gets or sets the custom domain name assigned to the storage account. Name is the CNAME source.')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedStorageEncryption', Mandatory, HelpMessage='Gets or sets the custom domain name assigned to the storage account. Name is the CNAME source.')]
+        [Parameter(ParameterSetName='UpdateViaIdentityExpandedKeyVaultEncryption', Mandatory, HelpMessage='Gets or sets the custom domain name assigned to the storage account. Name is the CNAME source.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Body')]
+        [System.String]
+        # Gets or sets the custom domain name assigned to the storage account. Name is the CNAME source.
+        ${CustomDomainName},
```

Comment:
This parameter cannot be mandatory, and it cna only be in parameter sets that include encryption

Reply:
Ditto